doc/doxygen/CMakeLists.txt

```diff
@@ -72,12 +72,9 @@ IF(DEAL_II_DOXYGEN_USE_MATHJAX)
   IF(MATHJAX_PATH AND NOT DEAL_II_DOXYGEN_USE_ONLINE_MATHJAX)
     SET(_mathjax_relpath ${MATHJAX_PATH})
     MESSAGE(STATUS "Using local MathJax: " ${MATHJAX_PATH})
-    SET(MATHJAX_RELPATH_CONFIG "MATHJAX_RELPATH = ${MATHJAX_PATH}")
```

Comment:
This is probably beyond my knowledge of this part of the cmake scripts, but why do you remove this line?